### PR TITLE
Change render_text to render( text => for sub postupdate in Server.pm

### LINKED ISSUE #1238: TO /update/<hostname> sends no response
When you POST valid data to the route above, Traffic Ops just closes the TCP connection, it doesn't actually send back a response. However, Traffic Ops does seem to successfully process the data that was posted.

### PR DESCRIPTION
**Pull request by hbeatty**
* traffic_ops/app/lib/UI/Server.pm

This should fix the 'no reply' and 500 error that was being returned when trying to clear the update column in TO.

fixes #1238

### CODE REVIEW

File: traffic_ops/app/lib/UI/Server.pm [hunk 1]
```diff
@@ -943,13 +943,13 @@ sub postupdate {
 	my $updated   = $self->param("updated");
 	my $host_name = $self->param("host_name");
 	if ( !&is_admin($self) ) {
-		$self->render_text( "Unauthorized.", status => 401, layout => undef );
+		$self->render( text => "Unauthorized.", status => 401, layout => undef );
```
Comment (mitchell852) [inline traffic_ops/app/lib/UI/Server.pm]: Can you change the status codes returned and see if you still get the expected results? For this one return a 403 (forbidden)

File: traffic_ops/app/lib/UI/Server.pm [hunk 2]
```diff
@@ -959,8 +959,8 @@ sub postupdate {
 	# resolve server id
 	my $serverid = $self->db->resultset("Server")->search( { host_name => $host_name } )->get_column('id')->single;
 	if ( !defined $serverid ) {
-		$self->render_text(
-			"Failed request.  Unknown server",
+		$self->render(
```
Comment (mitchell852) [inline traffic_ops/app/lib/UI/Server.pm]: for this one return a 404

File: traffic_ops/app/lib/UI/Server.pm [hunk 3]
```diff
@@ -943,13 +943,13 @@ sub postupdate {
...
-		$self->render_text( "Unauthorized.", status => 401, layout => undef );
+		$self->render( text => "Unauthorized.", status => 401, layout => undef );
 		return;
 	}
 
 	if ( !defined($updated) ) {
-		$self->render_text(
-			"Failed request.  Must provide updated status",
+		$self->render(
+			text => "Failed request.  Must provide updated status",
```
Comment (mitchell852) [inline traffic_ops/app/lib/UI/Server.pm]: for this one return 400 (Bad request)

File: traffic_ops/app/lib/UI/Server.pm (hunk 1 above)
Comment (mitchell852) [inline traffic_ops/app/lib/UI/Server.pm]: and change the text to "Forbidden"

**Comment by hbeatty** [comment]
@mitchell852 All changes made as requested.

**Pull request merged**